pkg/discovery/worker/group_metrics_collector.go

```diff
@@ -115,3 +114,11 @@ func (collector *GroupMetricsCollector) getGroupName(etype metrics.DiscoveredEnt
 
 	return ownerTypeString, ownerString, nil
 }
+
+func groupKey(kind, ns, name string) string {
```

Comment:
I could be wrong, but is this function being used anywhere?

Reply:
same question

Reply:
Apologies, I did intend to remove it but I published this PR probably half asleep late last night.. :)
Updated!